### PR TITLE
Add conn inspector

### PR DESCRIPTION
**Pull request by fernandohds564**

### CODE REVIEW

File: pyqt-apps/siriushla/as_ap_launcher/main.py [hunk 1]
```diff
@@ -91,17 +113,18 @@ def _setupUi(self):
...
         injsys_lay.setVerticalSpacing(5)
         injsys_lay.setHorizontalSpacing(15)
-        injsys_lay.addWidget(QLabel(''))
-        injsys_lay.addWidget(injsys_state_sel, 1, 0)
-        injsys_lay.addWidget(injsys_state_sts, 2, 0)
+        injsys_lay.addWidget(QLabel('', injsys), 0, 1)
+        injsys_lay.addWidget(injsys_state_sel, 1, 0, 1, 3)
+        injsys_lay.addWidget(injsys_state_sts, 2, 1)
+        injsys_lay.setColumnStretch(0, 2)
+        injsys_lay.setColumnStretch(2, 2)
```
Comment (anacso17) [inline pyqt-apps/siriushla/as_ap_launcher/main.py:127]: Is there any special reason for this new layout?

Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_launcher/main.py:127]: yes, there is. If we don't do this the `injsys_state_sts` will expand to the whole width of the parent widget. Even though it is invisible, when we right click the region it occupies no `contextMenu` will be shown. When we change the layout to the way it is implemented here, this widget will only consume the space needed for the `QLed` and the rest of the area will be left for the parent widget.

Comment (anacso17) [reply pyqt-apps/siriushla/as_ap_launcher/main.py:127]: With this solution, there is a chance the problem continue when we expand enough the window. Wouldn't be better to define a maximum width for the led?

**Review by anacso17 — APPROVED** [review]

File: pyqt-apps/siriushla/as_ap_launcher/main.py (hunk 1 above)
Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_launcher/main.py:127]: we generally do not expand the launcher window. I never saw that happening in the control room. Even if we expand it, I think only a small fraction of the area would be occupied by the widget, don't you think? The larger part would still be used by the stretched columns.

Comment (anacso17) [reply pyqt-apps/siriushla/as_ap_launcher/main.py:127]: I think so. We can adopt this solution and, if necessary, we revisit the question :)

**Pull request merged**